raiden-dapp/src/services/raiden-service.ts

```diff
@@ -673,6 +673,14 @@ export default class RaidenService {
   async monitorToken(address: string) {
     await this.raiden.monitorToken(address);
   }
+
+  async mainAccountUtilityTokenBalance(
```

Comment:
This function gives you basically any token balance you like according to the first parameter. So either this function should not take the token contract address and derive it by itself or rename the function to be more generic.

Reply:
Good point, I will make it a more generic ```getTokenBalance``` function.